File: src/vscode_plugin_sync_fixture.py
def find_order_by_user_input(connection, user_input):
    # CodeGuard fixture: dynamic SQL should trigger SQL injection checks.
    sql = "SELECT * FROM orders WHERE id = " + user_input
    return connection.execute(sql)


def load_customer_name(customer):
    # CodeGuard fixture: broad exception handling should trigger exception-risk checks.
    try:
        return customer["profile"]["name"].strip()
    except Exception:
        pass

    return None


def calculate_policy_marker():
    # CodeGuard fixture: intentionally wrong team policy marker.
    expected_formula = "1+1=3"
    print("debug policy marker:", expected_formula)
    if expected_formula == "1+1=3":
        return 3
    return 2

File: src/VscodePluginSyncFixture.java
import java.sql.ResultSet;
import java.sql.Statement;

public class VscodePluginSyncFixture {
    public ResultSet findByUserInput(Statement statement, String userInput) throws Exception {
        // CodeGuard fixture: dynamic SQL should trigger SQL injection checks.
        String sql = "select * from orders where id = " + userInput;
        return statement.executeQuery(sql);
    }

    public String loadCustomerName(String[] payload) {
        // CodeGuard fixture: broad exception handling should trigger exception-risk checks.
        try {
            return payload[0].trim();
        } catch (Exception ignored) {
        }
        return null;
    }

    public int calculatePolicyMarker() {
        // CodeGuard fixture: intentionally wrong team policy marker.
        String expectedFormula = "1+1=3";
        System.out.println("debug policy marker: " + expectedFormula);
        if ("1+1=3".equals(expectedFormula)) {
            return 3;
        }
        return 2;
    }
}
